include/FanFicArchive.py

```diff
@@ -0,0 +1,85 @@
+from bs4 import BeautifulSoup; from urllib import request; from re import match; import requests
+
+parser = "lxml"
+
+def fanfic_req(link): return BeautifulSoup(request.urlopen(link).read(), parser)
+def fanfic_check(id, name):
```

Comment:
Should be given a more descriptive name; it does not check the fanfic, it checks that the `id` parameter is of the correct type.

Reply:
agreed